.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: python
+env:
+  - PYTHON_VERSION=3.5
+before_install:
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+install:
+  - conda create --yes -n test-env python=$PYTHON_VERSION nose flake8 numpy
+  - source activate test-env
+  - pip install https://github.com/qiime2/qiime2/archive/master.zip https://github.com/qiime2/q2-types/archive/master.zip https://github.com/qiime2/q2templates/archive/master.zip https://github.com/biocore/emperor/archive/new-api.zip
```

Comment:
Instead of installing the `new-api` branch of Emperor, should we be installing the latest compatible Emperor release since it is a plugin dependency (e.g. like numpy, scikit-bio, etc)? @ebolyen, @thermokarst thoughts?